### DISCUSSION TITLE
Add BlockMerge script extension?

### DISCUSSION THREAD

**Discussion by LoFiApostasy** [bbc-mc/sdweb-merge-block-weighted-gui#24] [Show and tell]
This super handy script allows for running merges and generating images from memory, thus speeding up discovery and saving one's hard drive.  All credit goes to the original coder, but the VAE selection was broken and my sanity needed this script working so I fixed it.
Link: https://github.com/LoFiApostasy/block-merge-script

However having ZERO python coding experience, I was unable to implement some of the changes made to the blockmerge extension, I did however include some handy examples in the readme.  This would really help round out your extension if someone other than me, aka. complete hack, would keep it updated.

**Comment by Deph1** [comment]
is it possible to run an XYZ script that on each iteration it will change the values as per the XYZ instructions inputted?
